test/gui/tst_addAccount/test.feature

```diff
@@ -47,24 +47,24 @@ Feature: adding accounts
         Then credentials wizard should be visible
 
 
-    Scenario: Adding account with virtual files enabled through advance configuration
+    Scenario: Adding account with vfs enabled
         Given the user has started the client
-        When the user adds the first account with advanced configuration
+        And the user has added the following account information:
             | server   | %local_server% |
             | user     | Alice          |
             | password | 1234           |
-        And the user selects virtual_files option in advanced section
-        And the user selects enable_experimental_placeholder_mode option in enable experimental feature dialogue box
+        When the user selects vfs option in advanced section
+        And the user confirms the enable experimental vfs option
```

Comment:
I think select fit more than confirm

Reply:
This step can have two variations (`confirms` and `cancels`)